app/views/admin/project_steps/_show.html.slim

```diff
@@ -25,17 +48,19 @@
               = t("project_step.status.completed")
         - else
           = t("project_step.status.not_completed")
-      .actions
-        a.action.edit-step-action href="#"
-          i.fa.fa-pencil
-        - if policy(step).duplicate?
-          a.action.duplicate-step-action href="#"
-            i.fa.fa-files-o
+      - if context == "timeline"
+        .actions
+          a.action.edit-step-action href="#"
+            i.fa.fa-pencil
+            
+          - if policy(step).duplicate? && (context == 'timeline')
```

Comment:
Delete action may need to be available regardless of whether policy(step).duplicate? is true.